azext_iot/iothub/common.py

```diff
@@ -38,6 +72,30 @@
 NO_CHANGE_MSG = "Current Certificate Root Authority is already {0}. No updates are needed."
 
 
+class EndpointType(Enum):
+    """
+    Type of the message endpoint.
+    """
+    EventHub = 'eventhub'
```

Comment:
tiny nit to use consistent quoting, Python black formatting takes care of this for you.